Sources/App/commands.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import StevensonCore
+
+let fastlane: SlackCommand = (
+    help: """
+    Invokes specified lane on specified branch (or develop if not specified).
+    Provide options the same way as when invoking lane locally.
+
+    Example:
+        `/fastlane test_babylon branch:develop`
+    """,
+    token: ProcessInfo.processInfo.environment["SLACK_TOKEN_FASTLANE"]!,
```

Comment:
not sure if this will work right away when deployed, as I didn't try that yet. Maybe will need to change to launch arguments and Vapor's Environment type.